contributing/single-sourcing-content.md

```diff
@@ -219,20 +219,65 @@ To use the component at the beginning of a sentence, add a non-breaking space ch
 <Var name="dbt" /> is awesome!
 ```
 
-## Reusing snippets of content
+## Reusing content
 
-The Snippet component allows for content to be reusable throughout the Docs. This is very similar to the existing FAQ component.
+Use some techniques to share the same content on different pages, like partial files or snippets. 
+
+Using partial files, which is a built-in Docusaurus feature, is recommended over snippets.
+
+### Partial
+
+A partial file allows you to reuse content throughout the Docs. Here are the steps you can take to create and use a partial file:
+
+1. Create a new markdown partial file in the `website/snippets` directory. The file name must begin with an underscore, like `_filename.md`
+2. Go back to the Docs file that's pulling content from the partial file 
+3. Add the following import file: `import SetUpPages from '/snippets/_this-is-your-partial-file-name.md';`
+   * To use partials, you must always add an import file in that format. Note `SetUpPages` can be whatever makes sense for your purpose.
+   * `.md` needs to be added to the end of the filename. 
+3. Go to the next line and add `<SetUpPages anyname={'/snippets/_this-is-your-partial-file-name.md'}/>`. This fetches the reusable content in the partial file
```

Comment:
For this, step 2 looks correct with the import statement. 

For step 3, when using the partial component, you don't have to include the file path within a prop. The component can be used as `<SetUpPages />` without any props.

If you were passing in data into the partial, that is when we would use a prop on the component. This is useful when using dynamic data (for example if you wanted to use the same partial on multiple docs pages, but changing the values within the partial for each page).

Looking at the example in the [Docusaurus docs](https://docusaurus.io/docs/markdown-features/react#importing-markdown), if we wanted to pass in a `name` into the partial, we would do:

```
import PartialExample from '/snippets/_markdown-partial-example.mdx';

<PartialExample name="Sebastien" />
```

And within the partial file itself, the name prop would be available at `props.name`:
```
<span>Hello {props.name}</span>

This is demo content.
```